pkg/utils/utils.go

```diff
@@ -9,14 +9,9 @@ import (
 	"time"
 
 	"github.com/ikitiki/logical_backup/pkg/dbutils"
-	"github.com/ikitiki/logical_backup/pkg/message"
 )
 
-func TableDir(tbl message.NamespacedName, oid dbutils.Oid) string {
-	if oid == dbutils.InvalidOid {
-		panic(fmt.Sprintf("requested table directories for the table %s with invalid oid", tbl))
-	}
-
+func TableDir(oid dbutils.Oid) string {
```

Comment:
This will silently fail on many restore functions unless they are changed to provide a non-zero Did

Reply:
I've removed those changes